qiskit/transpiler/passes/synthesis/hls_plugins.py

```diff
@@ -263,7 +263,7 @@
       - Targeted connectivity
     * - ``"rustiq"``
       - :class:`~.PauliEvolutionSynthesisRustiq`
-      - use the synthesis method from https://github.com/smartiel/rustiq-core
+      - use the synthesis method from `this quantum circuit synthesis library <https://github.com/smartiel/rustiq-core>`_
       - all-to-all
```

Comment:
perhaps say:
this quantum circuit synthesis library --> rustiq circuit synthesis library